e2e/BugReporting.cs

```diff
@@ -0,0 +1,39 @@
+using Xunit;
+using Instabug.Captain;
+
+namespace E2E;
+
+public class BugReporting
+{
+  readonly Captain captain = new(
+      androidApp: Path.GetFullPath("../../../../example/build/app/outputs/flutter-apk/app-debug.apk"),

```

Comment:
Can we use absolute path instead of the relative one, since it's too confusing?
```suggestion
      androidApp: Path.GetFullPath("~/project/example/build/app/outputs/flutter-apk/app-debug.apk"),

```

Reply:
That would work on CI but it won't work locally unless we put the repo in `~/project` which isn't convenient